posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,816 @@
+// // // // // // // //
+
+Or if you're using link:{url-prefix}/docs/latest/container-images.html[container images]:
+
+[source]
+----
+FROM icr.io/appcafe/open-liberty:beta
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+If you're using link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[IntelliJ IDEA], link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Visual Studio Code] or link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse IDE], you can also take advantage of our open source link:https://openliberty.io/docs/latest/develop-liberty-tools.html[Liberty developer tools] to enable effective development, testing, debugging and application management all from within your IDE.
+
+For more information on using a beta release, refer to the link:{url-prefix}docs/latest/installing-open-liberty-betas.html[Installing Open Liberty beta releases] documentation.
+
+[#feedback]
+== We welcome your feedback
+
+Let us know what you think on link:https://groups.io/g/openliberty[our mailing list]. If you hit a problem, link:https://stackoverflow.com/questions/tagged/open-liberty[post a question on StackOverflow]. If you hit a bug, link:https://github.com/OpenLiberty/open-liberty/issues[please raise an issue].
+
+[#platform-ref]
+== Platforms and related versionless features reference list
+
+The following platform features are available in this beta release. You can specify up to 2 platform elements, one for MicroProfile, another for either Jakarta EE or Java EE.
+
+=== MicroProfile
+
+Platforms:
+
+* `microProfile-1.2`
+* `microProfile-1.3`
+* `microProfile-1.4`
+* `microProfile-2.0`
+* `microProfile-2.1`
+* `microProfile-2.2`
+* `microProfile-3.0`
+* `microProfile-3.2`
+* `microProfile-3.3`
+* `microProfile-4.0`
+* `microProfile-4.1`
+* `microProfile-5.0`
+* `microProfile-6.0`
+* `microProfile-6.1`
+
+MicroProfile versionless features:
+
+* `mpConfig`
+* `mpFaultTolerance`
+* `mpHealth`
+* `mpJwt`
+* `mpMetrics`
+* `mpOpenAPI`
+* `mpOpenTracing`
+* `mpRestClient`
+* `mpTelemetry`
+
+
+==== Jakarta EE
+
+Platforms:
+
+* `jakartaee-9.1`
+* `jakartaee-10.0`
+
+Jakarta EE versionless features:
+
+* `appAuthentication`
+* `appAuthorization`
+* `appClientSupport`
+* `appSecurity`
+* `batch`
+* `beanValidation`
+* `cdi`
+* `concurrent`
+* `connectors`
+* `connectorsInboundSecurity`
+* `data`
+* `enterpriseBeans`
+* `enterpriseBeansHome`
+* `enterpriseBeansLite`
+* `expressionLanguage`
+* `enterpriseBeansPersistentTimer`
+* `enterpriseBeansRemote`
+* `faces`
+* `j2eeManagement`
+* `jdbc`
+* `jsonb`
+* `jsonp`
+* `mail`
+* `managedBeans`
+* `mdb`
+* `messaging`
+* `messagingClient`
+* `messagingSecurity`
+* `messagingServer`
+* `pages`
+* `persistence`
+* `restfulWS`
+* `restfulWSClient`
+* `servlet`
+* `websocket`
+* `xmlBinding`
+* `xmlWS`
+
+==== Java EE
+
+Platforms:
+
+* `javaee-6.0`

```

Comment:
```suggestion
```

@cbridgha or @rsherget since this is an Open Liberty beta and Open Liberty doesn't support Java EE 6, should it be removed from the list?

Reply:
I'm not 100% sure on this answer but that sounds right to me.

Reply:
Actually we did not mark jsp-2.2 for the beta in OL since it was javaee-6.0, so yes `javaee-6.0` should be removed.